project/cmake/addons/depends/common/tinyxml/CMakeLists.txt

```diff
@@ -14,6 +14,11 @@ add_definitions(-DTIXML_USE_STL)
 
 add_library(tinyxml ${SOURCES})
 
+
+IF(CMAKE_SYSTEM_PROCESSOR STREQUAL "x86_64")
```

Comment:
Are we sure that this condition doesn't match on a Windows x64 installation? Because `-fPIC` certainly doesn't work on Windows (and we only build 32bit there anyway). What about OSX32? Do we compile it on a 64bit OS?